satpy/tests/reader_tests/test_utils.py

```diff
@@ -344,6 +344,36 @@ def test_generic_open_filename(self, open_mock):
 
         assert mock_fn_open.read.called
 
+    @mock.patch('bz2.decompress')
+    def test_unzip_FSFile(self, bz2_mock):
+        """Test the FSFile bz2 file unzipping techniques."""
+        mock_bz2_decompress = mock.MagicMock()
+        mock_bz2_decompress.return_value = b'TEST_DECOMPRESSED'
+
+        # test zipped FSFile
+        mem_fs = MemoryFileSystem()
+        mem_file = MemoryFile(fs=mem_fs, path="{}test.DAT.bz2".format(mem_fs.root_marker), data=b"TEST")
+        mem_file.commit()
+        fsf = FSFile(mem_file)
+
+        segment = 3
+        segmentstr = str(segment).zfill(2)
+
+        new_fname = hf.unzip_FSFile(fsf, prefix=segmentstr)
+        assert mock_bz2_decompress.called
```

Comment:
The mock object has a `assert_called` method you could use without the `assert ` statement before it.